index.html

```diff
@@ -232,18 +244,18 @@ <h2>Skills</h2>
                 <h2 class="white">Contact me</h2>
                 <p class="white">If you have an application you are interested in developing, a feature that you need built or a project that needs coding. I’d love to help with it </p>
             </div>
-            <form id="form" action="https://formspree.io/f/mqkoqdwg" method="POST">
+            <form id="form" action="https://formspree.io/f/mknaqnpb" method="POST">
                 <ul>
                     <li>
                         <ul id="contactInGrid">
                             <li>
                                 <div class="input-div">
-                                    <input
+                                    <input 
                                         type="text"
                                         name="name"
                                         maxlength="30"
                                         required="true"
-                                        placeholder="Lorem Ipsum"
+                                        placeholder="Alexandra Loremipsum"
```

Comment:
- Please implement this part of the project requirement. "A user cannot submit the form if one or more of the fields is empty" by adding "required" to the input field and the textarea

Reply:
In this other observation .... Did you request, the required field?? :-(

Look my validation:

**First field**

 
![01](https://user-images.githubusercontent.com/86327867/228659966-7fe010b9-974e-4c81-808f-0685d9707689.jpg)

**Second field**


![02](https://user-images.githubusercontent.com/86327867/228660035-5f3f4e04-fc84-443c-bd42-7897fdd7d260.jpg)


**Third field**

![03](https://user-images.githubusercontent.com/86327867/228660118-ea862a0d-cb13-4ab4-853f-8f9c91cad8aa.jpg)

As you can see, dear @Eternalgratis I guessed that I did my job. If I'm wrong, please tell me where and I'll do the changes.

Kind regards, 

Javier.-

Reply:
Hi @jlcarrascof ,
Apologies for this review. There was a mix-up somewhere. Kindly resubmit for it to get approved